src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1134,6 +1138,36 @@ protected function get_existing_ids_from_import_data( $import_data ) {
 		return $existing_ids;
 	}
 
+	/**
+	 * Gets all ids for events that have been ignored
+	 *
+	 * @param array $check Array of event IDs to check
+	 *
+	 * @return array
+	 */
+	protected function get_ignored_ids( $check ) {
+		if ( empty( $check ) ) {
+			return array();
+		}
+
+		$first = reset( $check );
+
+		if ( ! is_numeric( $first ) ) {
+			$check = wp_list_pluck( $check, 'post_id' );
```

Comment:
It would be cool wit we could pluck from an array of `WP_Post` too. What do you think?